substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/SubstrateOptions.java

```diff
@@ -435,6 +435,9 @@ public static boolean useLLVMBackend() {
     @Option(help = "file:doc-files/UseMuslCHelp.txt", type = OptionType.Expert)//
     public static final HostedOptionKey<String> UseMuslC = new HostedOptionKey<>(null);
 
+    @Option(help = "When set to true, builds a native image using Bionic as the libc implementation", type = OptionType.Expert)//
```

Comment:
We should explicitly state that this does actually download and link against bionic libraries yet, but is used as a workaround to be able to do so externally.
We can change it to: "When set to true, sets the internally used libc to Bionic. Note that this does not currently download and link against Bionic libc, but serves as a workaround that makes it possible externally"

Reply:
Fixed.